packages/ui/src/components/ChatMessage/ChatContent.tsx

```diff
@@ -55,9 +55,15 @@ function ShipMention({ ship }: { ship: string }) {
 export function InlineContent({
   story,
   color = '$primaryText',
+  isGalleryPost = false,
```

Comment:
Proposal for this + the ref `inGalleryPost` param: how about we use `viewMode`? Which could be `'chat' | 'block' | 'note'. Since we'll probably need similar switches for note, and maybe feed etc. as time goes on.

Reply:
Good point